lib/defaults.js

```diff
@@ -35,12 +37,12 @@ module.exports = {
     function createPackage(name, tag) {
       return {
         ordId: `${regexWithRemoval(name)}:package:${capNamespace}${tag}`,
-        title: `sample title for ${regexWithRemoval(name)}`,
-        shortDescription: "Here is the shortDescription for packages",
-        description: "Here is the description for packages",
+        title: nameWithSpaces(name),
+        shortDescription: "Description for " + nameWithSpaces(name),
```

Comment:
```suggestion
        shortDescription: "Short description for " + nameWithSpaces(name),
```

Reply:
Maybe if we explicitly mention the prefixes like that, it motivates people to look into where to change it, once they see it published / exposed somewhere. Then it would be good to see that one is the short description and the other is the (full) description.